scripts/west_commands/runners/openocd.py

```diff
@@ -29,6 +27,19 @@
 DEFAULT_OPENOCD_RESET_HALT_CMD = 'reset init'
 DEFAULT_OPENOCD_TARGET_HANDLE = "_TARGETNAME"
 
+
+def to_num(number):
+    dev_match = re.search(r"^\d*\+dev", number)
+    dev_version = not dev_match is None
```

Comment:
Since you're touching this

```suggestion
    dev_version = dev_match is not None
```

Reply:
Fixed!

Reply:
Not anymore